Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -799,9 +799,10 @@ def license(license_path):
 def com_google_fonts_check_family_has_license(licenses):
   """Check font has a license."""
   if len(licenses) > 1:
+    filenames = [os.path.basename(f) for f in licenses]
     yield FAIL, Message("multiple",
                         ("More than a single license file found."
-                         " Please review."))
+                         " [{}] were found.".format(", ".join(filenames))))
```

Comment:
Here I would suggest using `f"More than a single license file found: {pretty_print_list(filenames)}"`

This helper function is available via `from fontbakery.utils import pretty_print_list`

Reply:
Thanks, I'm now using the helper function.